### PR TITLE
Write helm command to file for api aggregation

### PR DESCRIPTION
**Pull request by jpeeler**
Since we're already writing out all the certificates, might as well write out
the helm command with the arguments already filled out too. This is clearly an
optional commit, but while testing this I used the script helper several times.

### CODE REVIEW

File: contrib/svc-cat-apiserver-aggregation-tls-setup.sh [hunk 1]
```diff
@@ -59,3 +61,16 @@ echo '{"CN":"'${SVCCAT_SERVICE_NAME}'","hosts":['${ALT_NAMES}'],"key":{"algo":"r
 export SC_SERVING_CA=${SVCCAT_CA_CERT}
 export SC_SERVING_CERT=apiserver.pem
 export SC_SERVING_KEY=apiserver-key.pem
+
+cat <<EOF >> $HELM_SCRIPT
```
Comment (MHBauer) [inline contrib/svc-cat-apiserver-aggregation-tls-setup.sh]: one `>` redirect. otherwise helm-install.sh will get bigger and bigger each time.

**Comment by jpeeler** [comment]
Ah, right. Will update.

**Comment by jpeeler** [comment]
The previouis jenkins run failed in: https://service-catalog-jenkins.appspot.com/job/service-catalog-PR-testing2/1860/console. Looking to see if increasing the timeout fixes the problem.

**Comment by nilebox** [comment]
@jpeeler This change has broken the docs for installing Service Catalog on Mac OS: [docs/install-1.7.md#step-1---generate-tls-certificates](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/install-1.7.md#step-1---generate-tls-certificates).

See `Step 2` on this page which provides different commands for running `helm install` on Linux and Mac OS (different `base64` command format).
So we need to whether detect OS, or split the script into two - one of them shared between docs and full script, and another being a full Linux-specific script wrapper with `helm install`.

**Comment by jpeeler** [comment]
@nilebox see my proposal here: https://github.com/kubernetes-incubator/service-catalog/pull/1185#issuecomment-326446229

**Pull request merged**